test/data_bundle_test.dart

```diff
@@ -1,4 +1,5 @@
 @TestOn('browser')
+@Timeout(Duration(minutes: 2))
```

Comment:
Yes, it can take a lot of time to generate the binaries in the browser before starting the test. I noticed that the github action was timing out some times and I added this timeout.